conda/recipes/libcugraph/meta.yaml

```diff
@@ -28,10 +28,12 @@ requirements:
     - libcudf=0.7*
     - cython
     - cudatoolkit {{ cuda_version }}.*
+    - boost
```

Comment:
Is `boost` the same as `boost-cpp`?  I see the library in question provided by `boost-cpp` on another system:
```
/conda/pkgs/boost-cpp-1.68.0-h11c811c_1000/lib/libboost_system.so.1.68.0
```

Reply:
@rlratzel I went with `boost` because that's what is in the conda environment files.
https://github.com/rapidsai/cugraph/blob/branch-0.8/conda/environments/cugraph_dev.yml#L22

Reply:
sounds good to me.